pkg/distributors/test/MultiRewards.test.ts

```diff
@@ -61,6 +76,29 @@ describe('Staking contract', () => {
         false
       );
     });
+
+    it('reverts if a random user attempts to allowlist themselves', async () => {
+      await expect(
+        stakingContract.connect(other).allowlistRewarder(pool.address, rewardToken.address, other.address)
+      ).to.be.revertedWith("only accessible by governance, pool or it's asset managers");
+    });
+
+    it('allows the vaults authorizer to allowlist rewarders after changing', async () => {
+      let action = await actionId(vault, 'setAuthorizer');
+      await authorizer.connect(admin).grantRole(action, admin.address);
+
+      // set new authorizer on the vault
+      const newAuthorizer = await deploy('v2-vault/Authorizer', { args: [other.address] });
+      await vault.connect(admin).setAuthorizer(newAuthorizer.address);
+
+      action = await actionId(stakingContract, 'allowlistRewarder');
+      await newAuthorizer.connect(other).grantRole(action, other.address);
+
+      // that authorizer allowlists a rewarder
+      await stakingContract.connect(other).allowlistRewarder(pool.address, rewardToken.address, lp.address);
+
+      expect(await stakingContract.isAllowlistedRewarder(pool.address, rewardToken.address, lp.address)).to.equal(true);
+    });
```

Comment:
I'm not sure how useful this test is. At the end of the day you're just testing that `MultiRewards` maintains sync with the Vault in terms of which Authorizer is uses so this is a duplicate of one of the tests above.

Reply:
That's fair. A bit lengthy, but intended as a sanity check.  Do you think I should just remove it?

Reply:
Yeah, I'd just nix it.